content/data/esign/estamp.mdx

```diff
@@ -0,0 +1,205 @@
+---
+sidebar_title: Overview
```

Comment:
Can we have the sidebar title for eStamp as `eStamp` as there will be two `Overview` sections in the sidebar?

<img width="377" alt="Screenshot 2024-05-08 at 11 41 35" src="https://github.com/SetuHQ/docs/assets/9695866/4a4287da-8c3c-4b43-b8d3-a4cf0d9095d5">

Reply:
Updated in the subsequent PR, closing this